qiita_db/data.py

```diff
@@ -416,6 +416,17 @@ def _remove_filepath(self, fp, conn_handler, queue):
             self._set_link_filepaths_status("failed: %s" % msg)
             raise QiitaDBError(msg)
 
+        # The filepath belongs to one or more studies
+        studies_linked = self.studies
```

Comment:
I don't think this check should be here. This is avoiding to unlink any files from the raw data if the raw data is being used by another study. Is this the desired behavior?
If so, there is no mechanism in place to remove a raw data from a study; so once a raw data has been shared, you can add as many files as you want, but you cannot unlink them... I think this is a weird behavior.